admin/grants.sql

```diff
@@ -265,15 +265,7 @@ GRANT IMPORTED PRIVILEGES ON DATABASE IP_INFO
 TO ROLE DATA_ANALYTICS;
 
 -- Sage database privileges
-GRANT ALL PRIVILEGES ON FUTURE SCHEMAS IN DATABASE SAGE
-TO ROLE DATA_ENGINEER;
-GRANT ALL PRIVILEGES ON FUTURE TABLES IN DATABASE SAGE
-TO ROLE DATA_ENGINEER;
-GRANT CREATE SCHEMA, USAGE ON DATABASE SAGE
-TO ROLE DATA_ENGINEER;
-GRANT ALL PRIVILEGES ON ALL SCHEMAS IN DATABASE SAGE
-TO ROLE DATA_ENGINEER;
-GRANT ALL PRIVILEGES ON ALL TABLES IN DATABASE SAGE
+GRANT USAGE ON DATABASE SAGE
```

Comment:
Should we be explicitly revoking privileges from `DATA_ENGINEER` as well?

Reply:
> ```GRANT ALL PRIVILEGES ON ALL SCHEMAS IN DATABASE SAGE TO ROLE DATA_ENGINEER;```
I'm just concerned with the CITATIONS schema at the moment. Cleaning up the rest of the SAGE database is outside the immediate scope.